code/modules/research/xenobiology/crossbreeding/unstable.dm

```diff
@@ -0,0 +1,283 @@
+		human_thrower = thrower
+	return ..()
+
+
+/obj/item/slimecross/unstable/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(human_thrower)
+		on_effect(hit_atom)
+	qdel(src)
+
+/obj/item/slimecross/unstable/proc/on_effect(atom/hit_atom)
+	return
+
+/obj/item/slimecross/unstable/grey
+	colour = "grey"
+	effect_desc = "When thrown creates a rabid slime that is loyal to you."
+
+/obj/item/slimecross/unstable/grey/on_effect(atom/hit_atom)
+	var/mob/living/simple_animal/slime/agrro/slimey =  new /mob/living/simple_animal/slime/agrro(get_turf(hit_atom))
+	slimey.Friends[human_thrower] = SLIME_FRIENDSHIP_ATTACK
+	if(ishuman(hit_atom))
+		var/mob/living/carbon/human/humie = hit_atom
+		humie.buckle_mob(slimey)
+
+/obj/item/slimecross/unstable/orange
+	colour = "orange"
+	effect_desc = "When thrown creates a spot of immense heat"
+
+/obj/item/slimecross/unstable/orange/on_effect(atom/hit_atom)
+	new /obj/effect/hotspot(hit_atom.drop_location(),200,10000)
+
+/obj/item/slimecross/unstable/purple
+	colour = "purple"
+	effect_desc = "When thrown creates a smoke cloud of healing reagents."
+
+/obj/item/slimecross/unstable/purple/on_effect(atom/hit_atom)
+	var/datum/effect_system/smoke_spread/chem/S = new()
+	var/turf/location = get_turf(src)
+
+	// Create the reagents to put into the air
+	create_reagents(20)
+	reagents.add_reagent(/datum/reagent/medicine/regen_jelly,20)
+
+	// Attach the smoke spreader and setup/start it.
+	S.attach(location)
+	S.set_up(reagents, 3, location, silent = TRUE)
+	S.start()
+
+/obj/item/slimecross/unstable/blue
+	colour = "blue"
+	effect_desc = "When thrown extinguishes mobs and cools them down."
+
+/obj/item/slimecross/unstable/blue/on_effect(atom/hit_atom)
+	for(var/i in 0 to 5)
+		var/obj/effect/particle_effect/water/W = new /obj/effect/particle_effect/water(get_turf(src))
+		W.Move(get_step(W,pick(GLOB.alldirs)))
+
+	for(var/mob/living/carbon/carbie in range(1,src))
+		carbie.fire_stacks = -2
+		carbie.ExtinguishMob()
+		carbie.set_bodytemperature(BODYTEMP_NORMAL)
+
+/obj/item/slimecross/unstable/metal
+	colour = "metal"
+
+/obj/item/slimecross/unstable/metal/on_effect(atom/hit_atom)
+	if(!iscarbon(hit_atom))
+		new /obj/structure/wage_cage(get_turf(hit_atom))
+		return
+
+	var/mob/living/carbon/carbie = hit_atom
+	var/obj/item/restraints/handcuffs/metal_cage/wagie_cagie = new /obj/item/restraints/handcuffs/metal_cage(hit_atom)
+	wagie_cagie.apply_cuffs(carbie,human_thrower)
+
+/obj/item/slimecross/unstable/yellow
+	colour = "yellow"
+	effect_desc = "When thrown causes a tesla zap and a small emp."
+
+/obj/item/slimecross/unstable/yellow/on_effect(atom/hit_atom)
+	tesla_zap(get_turf(hit_atom),1,2000)
+	empulse(get_turf(hit_atom), 0,2,TRUE)
+
+/obj/item/slimecross/unstable/darkpurple
+	colour = "dark purple"
+	effect_desc = "When thrown spawns extremely hot plasma."
+
+/obj/item/slimecross/unstable/darkpurple/on_effect(atom/hit_atom)
+	atmos_spawn_air("plasma=[50];TEMP=[10000]")
```

Comment:
10,000 is super super hot. Most plasma spawners use 1000, otherwise the spawn heat ends up being more deadly than the fire itself.